### PR TITLE
[BREAKING CHANGE] Phillips Hue remote manufacturer specific cluster handling

### PR DESCRIPTION
**Pull request by dmulcahey**
This PR adds support for 4 actions per button: press, hold, short release and long release. It needs https://github.com/home-assistant/core/pull/37172 in order to function. 

Fixes: #86 

Things still needed:

- [x] device trigger support
- [x] deduplicate code between remote versions

### CODE REVIEW

**Comment by dmulcahey** [comment]
@Adminiuga I changed the triggers to make these what we use since they are more robust. Should we mark this as a breaking change in HA since if anyone was using the old ones they will no longer work?

**Comment by Adminiuga** [comment]
Yes, should be marked as breaking then. Would it break all device triggers or only hue ones

**Comment by dmulcahey** [comment]
Just these 2 remotes

**Comment by Adminiuga** [comment]
Maybe still should mark it as breaking but with limited scope for these two remotes.

**Comment by dmulcahey** [comment]
@Adminiuga think if I merge this and cut a release that we can pull this and the corresponding HA change into the beta? Or should we wait?

**Comment by Adminiuga** [comment]
Since this is a breaking change, maybe should wait a bit

File: zhaquirks/philips/__init__.py [hunk 1]
```diff
@@ -19,3 +59,57 @@ class PhilipsOnOffCluster(CustomCluster, OnOff):
 
     attributes = OnOff.attributes.copy()
     attributes.update({0x4003: ("power_on_state", PowerOnState)})
+
+
+class PhilipsBasicCluster(CustomCluster, Basic):
+    """Philips Basic cluster."""
+
+    attributes = Basic.attributes.copy()
+    attributes.update({0x0031: ("philips", t.bitmap16)})
```
Comment (Adminiuga) [inline zhaquirks/philips/__init__.py:68]: @dmulcahey where this is coming from? isn't this attr supposed to be on `0x0406` Occupancy cluster id?

Comment (dmulcahey) [reply zhaquirks/philips/__init__.py:68]: https://github.com/Koenkk/zigbee-herdsman-converters/blob/69befbcce5d744cf5f81e1cdf6f59cd5bcb5917e/devices.js#L931

Comment (dmulcahey) [reply zhaquirks/philips/__init__.py:68]: It makes the notification on the manufacturer cluster work from what I could see. It’s like a mode switch similar to what the Opple and Lightify x4 remotes do

Comment (Adminiuga) [reply zhaquirks/philips/__init__.py:68]: ah, ok. Just didn't see this in https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Philips-Hue-sniffing#hue-motion-sensor and thought it was the same attribute as sensitivity on occupancy cluster

**Pull request merged**